pkg/apis/api/types.go

```diff
@@ -762,6 +762,130 @@ type RiskLevel struct {
 	Level int
 }
 
+type ComponentReportRequestReleaseOptions struct {
+	Release string
+	Start   time.Time
+	End     time.Time
+}
+
+type ComponentReportRequestTestIdentificationOptions struct {
+	Component  string
+	Capability string
+	TestID     string
```

Comment:
what is this? a number in string form? or an actual string (e.g., from tests.name in the sippy DB)?

Reply:
hopefully that is the same as openshift-gce-devel.ci_analysis_us.junit.test_id which looks like some string to help uniquely identify a test.